pkg/envoy/eds/response.go

```diff
@@ -47,7 +47,15 @@ func getEndpointsForProxy(meshCatalog catalog.MeshCataloger, proxyIdentity ident
 			log.Error().Err(err).Msgf("Failed listing allowed endpoints for service %s for proxy identity %s", dstSvc, proxyIdentity)
 			continue
 		}
-		allowedServicesEndpoints[dstSvc] = endpoints
+		var availableEndpoints []endpoint.Endpoint
```

Comment:
I wonder if it would be equivalent to directly use `allowedServicesEndpoints[dstSvc]` without first pushing into this new slice?